src/app/pages/home/rolodex/rolodex.page.html

```diff
@@ -1,35 +1,61 @@
-<ion-toolbar>
-   <ion-searchbar animated="false" placeholder="Search cards" [(ngModel)]="searchTerm"></ion-searchbar>
+<ion-toolbar class="header">
+  <ion-buttons slot="primary">
+    <ion-menu-button></ion-menu-button>
+  </ion-buttons>
+  <ion-searchbar animated="false" placeholder="Search cards" [(ngModel)]="searchTerm" *ngIf="!portraitCardView"></ion-searchbar>
 </ion-toolbar>
+<ion-searchbar animated="false" placeholder="Search cards" [(ngModel)]="searchTerm" *ngIf="portraitCardView"></ion-searchbar>
```

Comment:
Why the duplication?

Reply:
Two different views based on the the physical orientation of the phone. The search bar moves to the toolbar in landscape to make cards fully visible on the screen.

Reply:
Is this really how it works? Why would you need two elements? Why wouldn't the one element shift in position? Are we hiding/showing each element based on orientation? That seems like overkill if they naturally fill the width of the containing element/screen. I'll need to see this in action.

Reply:
Assuming this doesn't break compilation, I'd say this is resolved until we can figure out whether this is really valid during integration testing (dev).